### ISSUE TITLE
Date input missing styles

### ISSUE THREAD

**Issue by franciscohanna92** [KacosPro/bootscake#4]
Hey Carlos, thanks for the helpers! Very useful.

I have a input like:

```
<?php echo $this->BootsCakeForm->control('date', ['empty' => true, 'required' => false]); ?>
```

Where `date` is a column of type `date`.

The input renders:
![image](https://user-images.githubusercontent.com/12707758/62844454-dcae3a00-bc97-11e9-92ef-7b60ffcccaed.png)

Is this expected behavior?

**Comment by KacosPro** [comment]
Hi @franciscohanna92 I'm glad they have helped someone :D

Yes, that's the native input for CakePHP date, I didn't change their UI because it rendered three selects one in each row and it wasn't a very good UX, what I usually do is use a JS implementation to change it for a date picker and handle it on the back end

**Comment by franciscohanna92** [comment]
Yes, my solution was to use a HTML native date input. But I think the three inputs for cake are good UX and more compatible across diferente platforms.

Would you accept a PR fixing the styles of the inputs so they all render in one single row and with `form-control` styles?

**Comment by KacosPro** [comment]
Of course!

**Comment by franciscohanna92** [comment]
#5 there you go!

**Comment by KacosPro** [comment]
@franciscohanna92 sorry it took me this long to accept the PR, huge thanks!
The styles are available on v0.0.6
